src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -266,6 +266,11 @@ private static XmlNode SetElement(XmlDocument document, XmlNode node, string nam
             return fileContainer;
         }
 
+        private bool IsOnlyPresentInMaestroManagedFeed(HashSet<string> locations)
+        {
+            return locations != null && locations.All(l => IsMaestroManagedFeed(l));
```

Comment:
I think this will catch the azure devops artifact locations that we populate the artifacts with originally:

PS C:\enlistments> darc get-asset --name "Microsoft.DotNet.Maestro.Tasks" --version "1.1.0-beta.19530.2" --max-age 90
Looking up assets with name 'Microsoft.DotNet.Maestro.Tasks' and version '1.1.0-beta.19530.2' in the last 90 days
Microsoft.DotNet.Maestro.Tasks @ 1.1.0-beta.19530.2
Repository:    https://github.com/dotnet/arcade-services
Branch:        production
Commit:        a449ef723256ae11c95a4869ef70e756daaa7f36
Build Number:  20191030.2
Date Produced: 10/30/2019 8:08 AM
Build Link:    https://dev.azure.com/dnceng/internal/_build/results?buildId=407404
BAR Build Id:  33291
Released:      False
Channels:
- .NET Eng - Latest
Locations:
- https://dev.azure.com/dnceng/internal/_apis/build/builds/407404/artifacts (Container)
- https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json (NugetFeed)
- https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json (NugetFeed)

The right thing to do is probably is to only NuGetFeed locations, if that's not already the case.

Reply:
Hmm yeah, good catch. I'll also add a container like that to the scenario test

Reply:
Done, we only care about nugetFeed type locations for dependency updates now.

Added the scenario where we only have a container + a maestro managed feed, and the feed is flowing.

https://dev.azure.com/dnceng/internal/_git/maestro-test2/pullrequest/5476?_a=files&path=%2FNuGet.config